example-app/resources/views/cities/index.blade.php

```diff
@@ -0,0 +1,191 @@
+            <button id="add-city" class="bg-blue-500 text-white px-4 py-2 mt-2 rounded">Add city</button>
+        </div>
+    </div>
+@endsection
+
+@push('scripts')
+    <script>
+        $(document).ready(function() {
+            const urlParams = new URLSearchParams(window.location.search);
+            const currentAirlineId = urlParams.get('airline_id');
+
+            $('#add-city').on('click', function() {
+                const name = $('#city-name').val();
+                if (!name) {
+                    alert('The name of the city cant be empty.');
+                    return;
+                }
+
+                $.ajax({
+                    url: '{{ route('api.cities.store') }}',
+                    type: 'POST',
+                    data: {
+                        name: name,
+                        _token: '{{ csrf_token() }}',
+                    },
+                    success: function(response) {
+                        if (currentAirlineId) {
+                            alert(
+                                'The city has been added successfully.'
+                            );
+                        } else {
+                            $('#city-table').append(`
+                                <tr data-id="${response.city.id}">
+                                    <td class="py-2 px-4 border-b">${response.city.id}</td>
+                                    <td class="py-2 px-4 border-b">${response.city.name}</td>
+                                    <td class="py-2 px-4 border-b">0</td>
+                                    <td class="py-2 px-4 border-b">0</td>
+                                    <td class="py-2 px-4 border-b">
+                                        <button class="btn-edit bg-yellow-500 text-white px-3 py-1 rounded">Edit</button>
+                                        <button class="btn-delete bg-red-500 text-white px-3 py-1 rounded">Delete</button>
+                                    </td>
+                                </tr>
+                            `);
+                        }
+                        $('#city-name').val('');
+                    },
+                    error: function(xhr) {
+                        if (xhr.responseJSON && xhr.responseJSON.errors) {
+                            alert(xhr.responseJSON.errors.name[0]);
+                        } else if (xhr.responseJSON && xhr.responseJSON.message) {
+                            alert(xhr.responseJSON.message);
+                        } else {
+                            alert('Unkown error adding city.');
+                        }
+                    }
+                });
+            });
+
+            $('.city-container').on('click', '.btn-delete', function() {
+                const row = $(this).closest('tr');
+                const id = row.data('id');
+
+                $.ajax({
+                    url: `{{ url('/api/cities') }}/${id}`,
```

Comment:
nice use of url() function, altought you can use relative URLs ( just /api/citites) and it will work